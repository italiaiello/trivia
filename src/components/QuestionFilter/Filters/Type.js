import React from 'react'

const Type = ({ setType }) => {

    const onTypeChange = e => {
        setType(e.target.value)
    }

    return (
        <article className="filterContainer">
            <select id="categories" name="categories" onChange={onTypeChange}>
                <option value="Any">Any Type</option>
                <option value="multiple">Multiple Choice</option>
                <option value="boolean">True / False</option>
            </select>
        </article>
    )
}

export default Type