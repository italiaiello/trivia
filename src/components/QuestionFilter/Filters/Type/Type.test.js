import React from 'react'
import { shallow } from 'enzyme'
import Type from './Type'

it('checks if it Type.js renders properly', () => {

    expect(shallow(<Type />)).toMatchSnapshot()
})