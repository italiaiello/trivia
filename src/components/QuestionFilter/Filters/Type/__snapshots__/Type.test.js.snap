// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`checks if it Type.js renders properly 1`] = `ShallowWrapper {}`;
